.github/workflows/test.yml

```
@@ -25,3 +25,10 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true # runs bundle install and caches installed gems automatically
       - run: bundle exec rake
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: standardrb
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: amoeba/standardrb-action@v4
```

This action should be able to annotate failures inline, but for some reason it's not failing even with a test commit I added to try to force it to. (it does locally.) I'll try to into it some more, but I might just fallback to running `bundle exec standardrb` manually and let the build fail without annotations for now.